tests/models/resource_test.py

```diff
@@ -352,7 +352,8 @@ def test_delete_acts_on_custom_indices(self, mock):
         other_resource = Resource(pk=uuid.uuid4())
         with sync_overridden_test_settings_to_arches():
             self.test_resource.delete_index(other_resource.pk)
-        self.assertIn(str(other_resource.pk), str(mock._mock_call_args))
+        # delete_resources() was called with the correct resource id.
+        self.assertEqual(other_resource.pk, mock._mock_call_args[1]["resources"].pk)
```

Comment:
Completely unrelated, but I had this lying around and didn't want to forget about it. When I wrote this test the assertion didn't really capture the issue.